### ISSUE TITLE
Add Slim syntax support

### ISSUE THREAD

**Issue by Defman21** [Komodo/KomodoEdit#1834]
### Short Summary

title.

## [Slim](http://slim-lang.com/)

**Comment by Defman21** [comment]
I also want to ask: would Komodo support something like:
```slim
html
  head
    javascript:
      alert("Hello, world!")
```

I mean `javascript:` thing: everything after that should be highlighted as JS (depends on the indentation). Also there is `ruby:` which works in the same way.

**Comment by mitchell-as** [comment]
I do not believe supporting that syntax is possible with Komodo's UDL lexers. We'd have to write our own lexer for this, which would pretty much be a parser. Embedding JS and Ruby into it would require writing sublexers -- very time consuming.

**Comment by Defman21** [comment]
Hmm okay, that's not that necessary. Once there will be a Slim parser, I'll try to write sublexers for it (there are some things like `<style>` or `<script>` in HTML which I believe works in the same way)
